tests/test_devices.py

```diff
@@ -255,6 +255,152 @@ def test_device_inventory_update_success(client, mocker):
+        device = AlarmDevice(client)
+        mocker.spy(device._connection, "query")
+        device.connect("username", "password")
+        inputs = {
+            0: "Entryway Sensor",
+            1: "Outdoor Sensor 1",
+            2: "Outdoor Sensor 2",
+        }
+        # Test
+        device.update()
+        device_inputs = {}
+        for input_id, name in device.inputs:
+            device_inputs.update({input_id: name})
+        assert device_inputs == inputs
+
+    def test_all_properties_empty(client, mocker):
+        """Should check store the e-connect System status in the inventory device object."""
+        device = AlarmDevice(client)
+        mocker.spy(device._connection, "query")
+        device.connect("username", "password")
+        # Test
+        device.update()
+        device._inventory = {}
+        device_inputs = list(device.inputs)
+        assert not device_inputs
+
+    def test_input_property_empty(client, mocker):
+        """Should check store the e-connect System status in the inventory device object."""
+        device = AlarmDevice(client)
+        mocker.spy(device._connection, "query")
+        device.connect("username", "password")
+        # Test
+        device.update()
+        device._inventory.update({"inputs": {}})
+        device_inputs = {}
+        device_inputs = list(device.inputs)
+        assert not device_inputs
+
+
+class test_sectors:
+    def test_property_populated(client, mocker):
+        """Should check if the device property are correctly populated"""
+        device = AlarmDevice(client)
+        mocker.spy(device._connection, "query")
+        device.connect("username", "password")
+        sectors = {
+            0: "S1 Living Room",
+            1: "S2 Bedroom",
+            2: "S3 Outdoor",
+        }
+        # Test
+        device.update()
+        device_sectors = {}
+        for sector_id, name in device.sectors:
+            device_sectors.update({sector_id: name})
+        assert device_sectors == sectors
+
+    def test_all_properties_empty(client, mocker):
+        """Should check store the e-connect System status in the inventory device object."""
+        device = AlarmDevice(client)
+        mocker.spy(device._connection, "query")
+        device.connect("username", "password")
+        # Test
+        device.update()
+        device._inventory = {}
+        device_sectors = list(device.sectors)
+        assert not device_sectors
+
+    def test_sectors_property_empty(client, mocker):
+        """Should check store the e-connect System status in the inventory device object."""
+        device = AlarmDevice(client)
+        mocker.spy(device._connection, "query")
+        device.connect("username", "password")
+        # Test
+        device.update()
+        device._inventory.update({"sectors": {}})
+        device_sectors = list(device.sectors)
+        assert not device_sectors
+
+
+class test_alerts_v2:
```

Comment:
As this is an accurate replica of previous tests, I think you should apply the same suggestions to this class too.